src/luascript.cpp

```diff
@@ -12060,9 +12061,9 @@ int LuaScriptInterface::luaMonsterTypeGetSummonList(lua_State* L)
 		return 1;
 	}
 
-	int index = 0;
-	lua_createtable(L, monsterType->summonList.size(), 0);
-	for (const auto& summonBlock : monsterType->summonList) {
+	int32_t index = 0;
```

Comment:
Revert this to int. It's only used to pass the index to lua_rawseti, which takes int as argument.